t/t2717-python-cli-plugins.t

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+test_description='Test command line plugin interface'
+
+. $(dirname $0)/sharness.sh
+
+export FLUX_CLI_PLUGINPATH=${FLUX_BUILD_DIR}/t/cli-plugins/
```

Comment:
This line is why distcheck is failing. The test plugins are not in the build directory, they are in the source directory.
```suggestion
export FLUX_CLI_PLUGINPATH=${SHARNESS_TEST_SRCDIR}/cli-plugins/
```